user-group/survey01-email.html

```diff
@@ -0,0 +1,113 @@
+<body style="margin:0;padding:0">
+  <div style="Margin:0;padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;min-width:100%;background-color:#EAECEC;">
+    <table style="border-spacing:0;width:100%;table-layout:fixed;">
+      <tbody>
+        <tr>
+          <td align="center" style="padding-top:30px;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;">
+
+            <center>
+
+              <table style="border-spacing:0;Margin:0 auto;" width="600">
+                <tbody>
+                  <tr>
+                    <td bgcolor="#111419" style="padding-top:60px;padding-bottom:30px;padding-right:0;padding-left:0;vertical-align:top;Margin:0 auto;text-align:center;" width="600">
+                      <h1>
+                        <img alt="Ally logo" height="102px" src="/assets/img/ally-logo.png" style="Margin-bottom:10px;" title="Ally" />
```

Comment:
Given that we'll be sending this as an email, the image links should be changed to absolute links (http://ally.ac/...)